src/deckgl-layers/layer-utils/cpu-aggregator.js

```diff
@@ -182,7 +182,8 @@ function getSubLayerAccessor(dimensionState, dimension, layerProps) {
     const cv = bin && bin.value;
     const domain = scaleFunc.domain();
 
-    const isValueInDomain = cv >= domain[0] && cv <= domain[domain.length - 1];
+    const isValueInDomain =
+      (cv >= domain[0] && cv <= domain[domain.length - 1]) || valueDomain.includes(cv);
```

Comment:
I am concerned about performance. this function is called in each render pass on all bins. if this is only to cover categorical value, a more sensible solution is to call deck.gl bin sorter to properly sort categorical values `string` etc so we can still use the `>=` comparison

Reply:
That's fair. I'll go ahead and extend BinSorter to use a better sort function.